packages/patch-editing--transactions/src/transaction-manager.ts

```
@@ -42,23 +53,40 @@ export class TransactionManager {
   commitTransaction(description?: string) {
     if (!this.hasTransaction) throw new Error('No open transaction found');
 
+    let isOutermost = this._transactionStack.size === 1;
```

ESLint

[error] 'isOutermost' is never reassigned. Use 'const' instead.

(prefer-const)